### PR TITLE
Fix Haml 5.0.0 compatibility

### PR DESCRIPTION
**Pull request by dvdbng**
Issue #224

### CODE REVIEW

**Comment by trotzig** [comment]
Thanks for the contribution @dvdbng! Would you mind also updating the haml 5 version in this file? 
https://github.com/brigade/haml-lint/blob/master/Appraisals
(which I believe will make sure we catch similar things in the future).

File: lib/haml_lint/tree/tag_node.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class TagNode < Node
     #
     # @return [Array<String>]
     def dynamic_attributes_sources
-      @value[:attributes_hashes]
+      @value.fetch(:attributes_hashes, [])
```
Comment (michaelherold) [inline lib/haml_lint/tree/tag_node.rb]: 🤔  Do you know if support for this was dropped or if it was moved? I worry that we'll miss some things if we just default this to an empty array.

Comment (dvdbng) [inline lib/haml_lint/tree/tag_node.rb]: @michaelherold true, seems like it was renamed (and it's data structure changed too?) https://github.com/haml/haml/commit/7c523f278a7c62eea249ea29c797bf29377e136e
Doing this fixed haml-lint for my codebase, but likely some checks are not working properly

**Comment by mensfeld** [comment]
Do you guys have any timeframe on releasing new version with that fixed? I can't use the github version unfortunately :(

**Comment by sds** [comment]
@mensfeld HAML-Lint [0.25.0](https://github.com/brigade/haml-lint/releases/tag/v0.25.0) adds support for HAML 5.

**Comment by mensfeld** [comment]
@sds  thank you!

**Pull request closed without merging**